code/game/objects/items/weapons/RCD.dm

```diff
@@ -164,6 +166,14 @@
 	w_class = ITEMSIZE_SMALL
 	origin_tech = list(TECH_MATERIAL = 2)
 	matter = list(DEFAULT_WALL_MATERIAL = 30000,"glass" = 15000)
+	var/remaining = 10
+
+/obj/item/weapon/rcd_ammo/large
+	name = "high-capacity matter cartridge"
+	desc = "Do not ingest."

```

Comment:
... What... what happens if you ingest it?

Reply:
Nothing?

Reply:
Well that's no fun. Why put a warning label if it's not dangerous?

Reply:
Remind me to make it possible to eat these, and also make them do horrible things.